src/engine_core/rendering/src/Vulkan/VkTypes.hpp

```diff
@@ -9,19 +9,34 @@
 #include <vulkan/vulkan.h>
 #include <magic_enum.hpp>
 #include <glm/vec4.hpp>
+#include <glm/mat4x4.hpp>
 
 
 // Stuff from vk_mem_alloc to avoid cyclical references
 struct VmaAllocation_T;
 using VmaAllocation = VmaAllocation_T *;
 
+struct Vertex {
+	glm::vec3 position;
```

Comment:
Since everything is already being initialized, why not initialize this variable?
something like `position = {} `or `position = {0.0f, 0.0f}`?